bad.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package exampletest;
```

Buf (1.55.1)

Files with package "exampletest" must be within a directory "exampletest" relative to root but were in directory ".".

(PACKAGE_DIRECTORY_MATCH)